frontend/src/components/ExistingSamples.vue

```diff
@@ -181,6 +193,14 @@
       <span style="color: var(--v-anchor-base); cursor: pointer" @click="openPmpSelectedEntries()">pMp Historical</span>
     </footer>
 
+    <footer v-if="selectAllChecked || selectAllIndeterminate">
```

Comment:
Why are there two footers in the page? The second one completely covers the first one and "Add PWG" and "Remove PWG" buttons are no longer there.